### ISSUE TITLE
"You don't have permission to access the database" when trying to get DB connection info

### ISSUE THREAD

**Issue by christophberger** [teableio/teable#594]
**Describe the bug**
When I try to obtain database information via API > Postgres Connection, I get the error message "You don't have permission to access the database".

**To Reproduce**

1. Go to a table
2. Click on `< / > API`
3. Select "Postgresql Connection"

**Expected behavior**
DB connection info should appear

**Screenshots**
<img width="515" alt="Screenshot 2024-05-07 at 23 44 20" src="https://github.com/teableio/teable/assets/3835890/dafe9214-430c-489f-a85d-4c3c8eaf2a25">

** Client (please complete the following information):**

- OS: macOS Sonoma
- Browser: Arc (Chrome-based)
- Version: 1.2.0

**Platform (Please tell us which deployment version you are using)**
docker-standalone

**Additional context**
I am able to access the database from DBeaver using the connection info in `dockers/examples/standalone/.env`. So I cannot see what permissions are missing.

**Comment by tea-artist** [comment]
Only users with creator permission can view and create db connections on the interface. You can check the permission registration of the current account in space. （and we will optimize this error message

**Comment by christophberger** [comment]
Thank you for explaining the reason. The point is, I have set up the docker standalone example by following the steps in the docs. I did not notice any advice about verifying or setting creator permissions. As the only user on the server, I would have expected to be a creator automatically. 

I cannot find any permission settings at space level.  How can I change the permissions to become creator?
